brickstrap.sh

```diff
@@ -416,6 +455,7 @@ case "${cmd}" in
         create-rootfs
         create-tar
         create-image
+        dump-info-hook
```

Comment:
No. The way I set it up, it's only called from the case/esac block which invokes the command sequences. So it's called either when you invoke brickstrap with the run-hooks command or with the all command but never twice in the same brickstrap command.